hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/OzoneContainer.java

```diff
@@ -590,4 +608,15 @@ public ReplicationServer getReplicationServer() {
     return replicationServer;
   }
 
+  public void compactDb() {
+    for (StorageVolume volume : volumeSet.getVolumesList()) {
+      HddsVolume hddsVolume = (HddsVolume) volume;
+      CompletableFuture.runAsync(hddsVolume::compactDb, compactExecutor);
+      // If set dbVolumeSet only need to execute the compact db once
+      if (dbVolumeSet != null) {
```

Comment:
Not fully understand this. For every HddsVolume, there will be one RocksDB. It's true even when dbVolume is configured.